docs/porting/psa/crypto.md

```diff
@@ -0,0 +1,21 @@
+<h2 id="crypto-port">Mbed Crypto</h2>
+
+For information about Mbed Crypto, please refer to [the Mbed Crypto repository](https://github.com/ARMmbed/mbed-crypto).
+
+## Testing
+Mbed OS currently provide two test for Mbed Crypto
+* Test for initialisation of the Mbed Crypto module
+* Test for entropy injection feature
+
+In order to run these tests make sure the following configuration is set in your target:
+1.  "extra_labels" contains the label "PSA". For example in [K64F](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L1451) and [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7694) 
+2.  "MBEDTLS_PSA_CRYPTO_C" macro is enabled. For example in [K64F](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L1454) and [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7697)

```

Comment:
@netanelgonen  should we go with exact label name as it is written in json file.
so it would be easier to lookup for the customer?
FUTURE_SEQUANA

Reply:
actually, the target name is FEATURE_SEQUANA_PSA, for the lookup the user can use the link. he doesn't need to look for it by himself... 
I don't care about changing it but, I think this is more readable.